src/machine/board_nrf52840.go

```diff
@@ -0,0 +1,56 @@
+//go:build nrf52840 || circuitplay_bluefruit
+// +build nrf52840 circuitplay_bluefruit
```

Comment:
You could perhaps add a few other boards as well that are based on the nrf52840 (reelboard, itsybitsy-nrf52840, clue, to name a few). That way they can more easily be added to the playground later.

Reply:
Done.